source/developer/contribution-guide.md

```diff
@@ -62,12 +43,14 @@ When submitting a PR, check that:
 2. PR title begins with the Jira Ticket ID (eg `PLT-394:`, [see examples](https://github.com/mattermost/platform/pulls?q=is%3Apr+is%3Aclosed))  
 3. PR comment describes the changes and how the feature is supposed to work  
 
+## Managing an Open Pull Request 
+
 After submitting a PR, before it is merged:  
 
 1. Automated build process must pass  
     1. If the build fails, check the error log to narrow down the reason  
     2. Sometimes one of the multiple build tests will randomly fail due to issues in Travis CI. If you see just one build failure and no clear error message, it may be a random issue. Add a comment so the reviewer for your change can re-run the build for you, or close the PR and re-submit and that typically clears the issue.  
 - PR must be reviewed by two team members, and reviewer feedback must be addressed  
-- UI must be approved by PM  
+- UI must be approved by PM
```

Comment:
I think we may need the two spaces at the end of a line to get it to render properly on docs.mattermost.com